packages/dashboard-v2/src/components/forms/SignUpForm.js

```diff
@@ -0,0 +1,109 @@
+import PropTypes from "prop-types";
+import { Formik, Form } from "formik";
+import * as Yup from "yup";
+
+import { TextField } from "../Form/TextField";
+import { Button } from "../Button";
+
+import accountsService from "../../services/accountsService";
+
+// TODO: better password strength validation
+const registrationSchema = Yup.object().shape({
```

Comment:
I haven't introduced any specific password policy at this point, as I think it's worth discussing.

The password policy on our current dashboard is pretty loose (allows stuff like `123456`) -- I was wondering if we want to introduce something more strict. Maybe something like [this](https://stackoverflow.com/questions/55451304/formik-yup-password-strength-validation-with-react)?

Reply:
No policy there on the server side, it just can't be empty.

I updated & unified the policies between `SignUpForm` and `ResetPasswordForm`: 6b733ab7